compiler/README.md

```diff
@@ -0,0 +1,56 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
+
+# Introduction
+
+This document discusses the ONE compiler. ONE compiler analyses, quantizes and optimizes the trained model. It takes many model formats and converts to a common 'Circle' format which is then deployed. This compiler runs on the target device and targets supported are ExynosXXX.
+
+<!-- Contents -->
+# Contents 
+* [Build Setup](#Build-Setup)
+* [Supported Formats](#Supported-Nerual-Network-Formats)
+* [Model Analysis](#Model-Analysis)
+* [Data Representation & Quantization](#Data-Representation-And-Quantization)
+* [Circle Format](#Circle-Format)
+* [Supported Targets](#Supported-Targets)
+
+<!-- Build Setup -->
+## Build Setup
+<!-- System Requirments -->
+* System Setup   
+This setup is to build the ONE compiler on Ubuntu 18.04.   
+Get the code [Samsung ONE](https://github.com/Samsung/ONE) from Github. 
+
+<!-- Software/Package Requirements -->
+* Software/Package Tested On
+    * Cmake (3.10.2)
+    * g++ toolchain (9.0.1)
+
+<!-- Build Configuration -->
+* Configure  
+In-Progress
+
+<!-- Build -->
+* Build  
+In-Progress
+
+* Build Issues
+
+## Supported Neural Network Formats
+Neural Networks can be held in many formats & ONE supports muliple formats too.  
+But the preferred being the TensorFlow Lite format. ONE provides tools to convert other formats to TensorFlow Lite (TFLite) and compiles/works on the TFLite and converts it to Circle format and fed to deployment.
+
+## Model Analysis
+
+## Data Representation And Quantization
+ONE supports both floating point and integer representation.
+ONE adopts a variety of quantizaton tecniques.
+
+## Optimization 
+What kind of optimizations are carried by NNCC?  
+Is it hardware specific? if so what kinds of hardware are supported?  
```

Comment:
There are no hardware specific optimizations (at least for now).
For now several fusion optimizations implemented (see [link](https://github.com/Samsung/ONE/blob/master/compiler/luci/pass/src/CircleOptimizer.cpp#L102))